gitlab/commodore-compile.jsonnet

```diff
@@ -46,10 +46,28 @@ local gitInsteadOf(cluster) =
     else if https_catalog != null then
       // set an insteadOf which injects credentials if we have a catalog URL
       // that's already HTTPS in Lieutenant.
+      local replacement = 'https://%(catalog_user)s:%(access_token)s@%(https_catalog)s' % {
+        catalog_user: cluster_access_user,
+        access_token: cluster_access_token,
+        https_catalog: https_catalog,
+      };
       [
-        'git config --global url."https://%(catalog_user)s:%(access_token)s@%(https_catalog)s".insteadOf https://%(https_catalog)s' % {
-          catalog_user: cluster_access_user,
-          access_token: cluster_access_token,
+        // Overide the original https:// URL for fetch and clone. Note that we
+        // use `git config set --append` here so we can configure two
```

Comment:
Guess this refers to the `--add` flag? Comment doesn't seem to match the code here.

Reply:
Yeah, this should be `git config --add` (side note: we can't actually use `git config set --append` because the Git version in the Commodore image is too old for that command)